src/style.ts

```diff
@@ -0,0 +1,34 @@
+import * as prebase from './prebase';
+
+export enum Enclosure {
+    // noinspection JSUnusedGlobalSymbols
+    RECTANGLE = 'rectangle',
+    SQUARE = 'square',
+    OVAL = 'oval',
+    CIRCLE = 'circle',
+    BRACKET = 'bracket',
+    TRIANGLE = 'triangle',
+    DIAMOND = 'diamond',
+    PENTAGON = 'pentagon',
+    HEXAGON = 'hexagon',
+    HEPTAGON = 'heptagon',
+    OCTAGON = 'octagon',
+    NONAGON = 'nonagon',
+    DECAGON = 'decagon',
+    NONE = 'none',  // special.  sets to undefined
+}
+
+
+// noinspection JSUnusedGlobalSymbols
+export class Style extends prebase.ProtoM21Object {
+    size: undefined;
```

Comment:
this one seems unnecessary to type. also, I looked in m21p and we never set or get it, so maybe we just jettison?

Reply:
Oops, forgot to define a value for it.  Will comment out for now, but want to remember that it should be there eventually.

Btw -- I've also rewritten Enclosure as a StrEnum in Python, but I think I need to make a v8 branch before committing it.